cmd/xl-v1-object.go

```diff
@@ -381,11 +381,11 @@ func (xl xlObjects) getObjectInfo(ctx context.Context, bucket, object string) (o
 	// get Quorum for this object
 	readQuorum, _, err := objectQuorumFromMeta(xl, metaArr, errs)
 	if err != nil {
-		return objInfo, toObjectErr(err, bucket, object)
+		return objInfo, err
```

Comment:
Why is `toObjectErr` removed here?

Reply:
@kannappanr its in the commit message 

>Also, make sure to not modify the underlying errors from
layers, we should return the error as is and one object
the layer should translate the errors.

Reply:
Sorry, missed it. Was it translated twice before?

Reply:
yes @kannappanr